deployments/registry.go

```diff
@@ -251,36 +298,9 @@ func (k Registry) apply(ctx context.Context, c *kubernetes.Cluster, ui *termui.U
 		return errors.Wrap(err, "failed trying to create the epinio API server cert")
 	}
 
-	ui.Success().Msg("Registry deployed")
-
-	log.Info("apply done")
-
-	return nil
-}
-
-func (k Registry) GetVersion() string {
```

Comment:
I just wish all the deployments had the same order for these funcs :)

Reply:
True but the files are so big that we rarely scroll. Search and code navigation does the trick most of the time. If I start moving things it will be hard to review the actual changes. I think this should happen (if at all) in a PR dedicated to moving code (no changes at all) to make reviewing easier.

Reply:
Yes, if we do it, it should be separate